drawings-gods_data-lists.ttl

```diff
@@ -5337,30 +5436,37 @@
 			knora-base:hasRootNode <http://rdfh.ch/lists/drawings-gods-2016-list-FiguresHList> ;
 			knora-base:listNodePosition 0 ;
 			rdfs:label "Figure humanoide"@fr , "humanoid figure"@en ;
-			knora-base:hasSubListNode <http://rdfh.ch/lists/drawings-gods-2016-list-FiguresHList-normalhuman> ,
+			knora-base:hasSubListNode <http://rdfh.ch/lists/drawings-gods-2016-list-FiguresHList--> ,
```

Comment:
Actually, I will let you modify this and then do a new commit. Then, I will merge.